src/main/java/com/company/model/MyCall.java

```diff
@@ -0,0 +1,112 @@
+package com.company.model;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+
+import javax.persistence.*;
+import java.sql.Timestamp;
+
+@Entity
+@Table(name = "calls")
+public class MyCall {
+    @Id
```

Comment:
why are your calls related to ONLY numbers? 
It's need related to User.
It's possible case:The user has changed his number, then user will not find his calls :))

Please, add user as relations.